### PR TITLE
add service registry access mgmt guide

### PR DESCRIPTION
**Pull request by smccarthy-ie**
Adds a short new guide on role-based access management using the Service Registry console

@EricWittmann @carlesarnal @wtrocki Please review for technical accuracy and let me know if you approve or have any comments, thanks.

### CODE REVIEW

File: access-mgmt-service-registry/README.adoc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+ifdef::context[:parent-context: {context}]
+:context: managing-access-service-registry
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a {registry} instance, administrator, or organization administrator in {registry-product-long}, you can manage the level of access that other user accounts and service accounts have to your {registry} instance. You can grant or remove access to your instance for specific user accounts in your organization based on user roles. You can also allow other users or service accounts to manage the level of access to your instance for you.
+
+{registry} instance owners or administrators can manage access for only the {registry} instances that they create or for instances that the owner has allowed them to access and change. Organization administrators can manage access for all {registry} instances.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
```
Comment (wtrocki) [inline access-mgmt-service-registry/README.adoc:57]: FYI - This problem is going to be solved soon (it was released in underlying library)

**Comment by wtrocki** [comment]
Looks good for me but I would like to have core team approving it.

**Comment by EricWittmann** [comment]
LGTM

File: access-mgmt-service-registry/README.adoc [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+=== Supported user roles in {registry-product-long}
+
+The {registry} web console provides an *Access* tab on the {registry} instance page. {registry} instance owners, administrators, and organization administrators can use this tab to manage the following user roles:
+
+* *Administrator* - Can view or write user roles and schema and API artifacts, configure global rules for compatibility and validity, and import/export {registry} data in this {registry} instance 
+* *Manager* - Can view or write schema and API artifacts in this {registry} instance
+* *Viewer* - Can view schema and API artifacts in this {registry} instance
+
+NOTE: The owner of a {registry} instance has the administrator role for that instance by default, and can assign roles in the same organization. Other users in the organization have read-only viewer access by default.
```
Comment (EricWittmann) [inline access-mgmt-service-registry/README.adoc:77]: The second sentence here is not true.  Other users in the organization have **no access** by default.

**Comment by EricWittmann** [comment]
Added a comment - nice catch @carlesarnal :)

**Review by EricWittmann — DISMISSED** [review]

File: access-mgmt-service-registry/README.adoc (hunk 2 above)
Comment (smccarthy-ie) [reply access-mgmt-service-registry/README.adoc]: @EricWittmann Good spot! Updated and promoted the NOTE to an IMPORTANT :)

**Comment by smccarthy-ie** [comment]
@rkubis @famartinrh Do you have any comments or approve from a QE perspective? Thanks.

**Comment by wtrocki** [comment]
Approving additional changes for CLI: f8dbf51801f2c0587933b35087f92df27ec95547

**Review by wtrocki — APPROVED** [review]

**Pull request merged**